sentinel-core/slot/global_slot_chain.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <memory>
+
+#include "sentinel-core/flow/flow_slot.h"
+#include "sentinel-core/slot/base/default_slot_chain_impl.h"
+#include "sentinel-core/slot/resource_node_builder_slot.h"
+#include "sentinel-core/slot/statistic_slot.h"
+
+namespace Sentinel {
+namespace Slot {
+
+static SlotChainSharedPtr BuildDefaultSlotChain() {
+  auto chain = std::make_shared<DefaultSlotChainImpl>();
+  chain->AddLast(std::make_unique<ResourceNodeBuilderSlot>());
+  chain->AddLast(std::make_unique<FlowSlot>());
+  chain->AddLast(std::make_unique<StatisticSlot>());
+  return chain;
+}
+
+static SlotChainSharedPtr GlobalSlotChain = BuildDefaultSlotChain();
```

Comment:
use `Utils:: Singleton` to build GlobalSlotChain, static variables are initialized order problem

https://isocpp.org/wiki/faq/ctors#static-init-order